BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -38,6 +38,42 @@ public struct AttributedLabel: Element, Hashable {
     /// Determines if the label should be included when navigating the UI via accessibility.
     public var isAccessibilityElement = true
 
+    /// A Boolean value that determines whether the label reduces the text’s font
+    /// size to fit the title string into the label’s bounding rectangle.
+    ///
+    /// Normally, the label draws the text with the font you specify in the font property.
+    /// If this property is true, and the text in the text property exceeds the label’s bounding rectangle,
+    /// the label reduces the font size until the text fits or it has scaled the font down to the minimum
+    /// font size. The default value for this property is false.
+    ///
+    /// If you change it to true, be sure that you also set an appropriate minimum
+    /// font scale by modifying the minimumScaleFactor property.
+    ///
+    /// This autoshrinking behavior is only intended for use with a single-line label.
+    public var adjustsFontSizeToFitWidth: Bool = false
+
+    /// The minimum scale factor for the label’s text.
+    ///
+    /// If the adjustsFontSizeToFitWidth is true, use this property to specify the
+    /// smallest multiplier for the current font size that yields an acceptable
+    /// font size for the label’s text.
+    ///
+    /// If you specify a value of 0 for this property, the label doesn't scale the text down.
+    /// The default value of this property is 0.
+    public var minimumScaleFactor: CGFloat = 0
```

Comment:
Nit: Should this be nullable instead of having a separate `adjustsFontSizeToFitWidth` property?

Seems fine to just match the UIKit APIs as you have, but I like having that connected state wrapped up in a single property since they're dependent on each other.

Reply:
I figured if we want the same properties, we may as well match them exactly, especially since there's no functional difference between someone assigning 0 vs nil.